### PR TITLE
#3025 - offering change request redirect

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Redirect on an offering change request approval back to the list of offering change requests.

### CODE REVIEW

File: sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue [hunk 1]
```diff
@@ -136,11 +136,7 @@ export default defineComponent({
           );
 
           router.push({
-            name: AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE,
-            params: {
-              programId: props.programId,
-              offeringId: props.offeringId,
-            },
+            name: AESTRoutesConst.REQUEST_CHANGE_OFFERINGS,
```
Comment (dheepak-aot) [inline sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue:139]: Can we also remove the obsolete route objects

route const `AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE` 

`AppRoutes.ViewOfferingChangeRequestComplete`

and the route itself.
image

**Review by lewischen-aot — APPROVED** [review]
Thanks for making the change @katrina-aot-git Nice work👍

**Review by sh16011993 — APPROVED** [review]
Nice Work @katrina-aot-git 👍

File: sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue (hunk 1 above)
Comment (katrina-aot-git) [reply sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue]: I have deleted this now.

File: sources/packages/web/src/router/AESTRoutes.ts [hunk 2]
```diff
@@ -539,18 +539,6 @@ export const aestRoutes: Array<RouteRecordRaw> = [
           clientType: ClientIdType.AEST,
         },
       },
-      {
-        path: AppRoutes.ViewOfferingChangeRequestComplete,
-        name: AESTRoutesConst.OFFERING_CHANGE_REQUEST_COMPLETE,
-        props: true,
-        components: {
-          default: ViewOfferingChangeRequestComplete,
```
Comment (dheepak-aot) [inline sources/packages/web/src/router/AESTRoutes.ts:547]: Please remove the obsolete variable and also the Vue file(ViewOfferingChangeRequestComplete.vue) from the file system

image

Comment (katrina-aot-git) [reply sources/packages/web/src/router/AESTRoutes.ts:547]: Also deleted, thanks for catching that too

**Review by dheepak-aot — APPROVED** [review]
Thanks for making the changes. Looks good 👍

**Pull request merged**